libursa/src/ffi/cl/prover.rs

```diff
@@ -1391,6 +1425,75 @@ mod tests {
         );
     }
 
+    #[test]
+    fn ursa_cl_prover_proof_builder_add_common_attribute_works() {
+        let (credential_pub_key, credential_priv_key, credential_key_correctness_proof) =
+            _credential_def();
+        let credential_values = _credential_values();
+        let credential_nonce = _nonce();
+        let (
+            blinded_credential_secrets,
+            credential_secrets_blinding_factors,
+            blinded_credential_secrets_correctness_proof,
+        ) = _blinded_credential_secrets(
+            credential_pub_key,
+            credential_key_correctness_proof,
+            credential_values,
+            credential_nonce,
+        );
+        let sub_proof_request = _sub_proof_request();
+        let credential_schema = _credential_schema();
+        let non_credential_schema = _non_credential_schema();
+        let credential_issuance_nonce = _nonce();
+        let (credential_signature, signature_correctness_proof) = _credential_signature(
+            blinded_credential_secrets,
+            blinded_credential_secrets_correctness_proof,
+            credential_nonce,
+            credential_issuance_nonce,
+            credential_values,
+            credential_pub_key,
+            credential_pub_key,
+        );
+        _process_credential_signature(
+            credential_signature,
+            signature_correctness_proof,
+            credential_secrets_blinding_factors,
+            credential_values,
+            credential_pub_key,
+            credential_issuance_nonce,
+            ptr::null(),
+            ptr::null(),
+            ptr::null(),
+        );
+        let proof_builder = _proof_builder();
+
+        let common_attr_name = string_to_cstring(String::from("master_secret"));
```

Comment:
This should be converted to a named static constant outside of the function.

```rust
static MASTER_SECRET: &'static str = "master_secret";
```

Reply:
Done